Unreal/Plugins/AirSim/Source/Vehicles/UrdfBot/MeshGeneration/StaticMeshGenerator.cpp

```diff
@@ -183,6 +183,7 @@ bool StaticMeshGenerator::CreateUnscaledMeshForLink(FString linkName, FString li
 
 TArray<TArray<FVector>> StaticMeshGenerator::CreateCollisionVAHCD(TArray<FVector> stlPoints, TArray<uint32> stlIndices, double concavity, unsigned int resolution, unsigned int maxNumVerticiesPerCH, double minVolumePerCh)
 {
+    TArray<TArray<FVector> > output;
```

Comment:
There seem to be a bunch of primarily cosmetic changes to this method. Apart from the delete[] at the end (which was a bug), can we revert the changes here to reduce churn? Or am I missing something?